xml/article_sap_trento.xml

```diff
@@ -293,9 +293,34 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         </para>
       </listitem>
     </itemizedlist>
-    <para> To install Trento on your &t.server; host proceed as follows: </para>
 
-    <procedure>
+    <para>
+      &t.server; premium is delivered as a Helm chart in order to facilitate its
+      installation process. If you already have a &k8s; cluster in place and
+      want to use it to run &t.server;, all you have to do is to install Helm
+      (<xref linkend="st-install-helm"/>) and then run command in <xref linkend="st-install-helm"/>
+      to deploy the helm chart in your &k8s; cluster.
+      If you don't have a &k8s; cluster or have one but don't want to use it for
+      Trento, you can deploy a small VM (see section 3.1 for minimum requirements
+      and follow steps 1 through 7 to get &t.server; up and running.
+    </para>
+    <para>
+      The &t.server; is available as package <package>trento-premium</package> in your
+      respective &suse; repository. The package does not require &productname;.
+      This package is available for customers that do not have a K3s; in place and
+      want to deploy the server on K3s for &productname;.
```

Comment:
LInes 308+309 have nothing todo with 310 and 311. Do you want to delete the whole paragraph?